### ISSUE TITLE
Notifications for deleted accounts

### ISSUE THREAD

**Issue by Semir1212** [humhub/humhub#5329]
Users whose accounts are deleted should be notified about the process. During the deletion process following options could be offered to the admins:

- Delete without sending notification
- 2-3 preset examples (e.g. violating community rules; insulting other users..) 
- A free text field for the admin to add an individual explanation

**Comment by luke-** [comment]
Wouldn't it suffice if we sent out a notification when a user account is "Suspended"?

**Comment by ArchBlood** [comment]
> Wouldn't it suffice if we sent out a notification when a user account is "Suspended"? 
> 

I agree here, although it should be possible for the admin to send a custom message instead of a generic notification.
